scenarios/kubernetes_bazel.py

```diff
@@ -71,7 +85,7 @@ def get_changed(base, pull):
         '--noshow_progress',
         'set(%s)' % diff).split('\n')
 
-def query(kind, selected_pkgs, changed_pkgs):
+def query(args, kind, selected_pkgs, changed_pkgs):
     """
```

Comment:
it is necessary to pass `args` through everywhere? should we just be passing `batch`, or something else more clever?

Reply:
It's not necessary but we usually do this in the scenarios. I'll try to make this cleaner.

Reply:
I wrapped up query/check/check_output for bazel in a class so we don't have to keep explicitly passing batch / args all over.